tools/analysis_tools/analyze_results.py

```diff
@@ -84,17 +89,22 @@ def __init__(self,
                  show=False,
                  wait_time=0,
                  score_thr=0,
-                 overlay_gt_pred=False):
+                 overlay_gt_pred=False,
```

Comment:
`DetLocalVisualizer` only support `overlay_gt_pred=True` now, see https://github.com/open-mmlab/mmdetection/blob/e3c0b6946f33ffefd6eb52b93ea5fd015dda4281/mmdet/visualization/local_visualizer.py#L378

So `overlay_gt_pred` may be unnecessary now.